dogechain-lab/dogechain
# [Feature] merge state update to single transaction

# Description

all trie state update merge in single exclusive transaction

# Changes include


## Testing


### Manual tests

environment:
* set up 4 validator network 
* use script to send multiple transactions

Test 1:
* stop a node randomly
* wait 1 minute
* restart stop node
* check whether node is synced

Test 2:
* kill a node randomly
* wait 1 minute
* restart killed node
* check whether node is synced
